src/sections/body/body.section.js

```diff
@@ -41,28 +41,25 @@ const Body = () => {
     }
   })
 
-  const [showAltForm, toggleAlt] = useState({ showAlt:false, meal:'', day:'', content:'' })
-
-  const toggleShowAltForm = (meal, day, content, bool) => {
-    toggleAlt({
-      showAlt: bool ? true : false,
-      meal:meal, day:day, content:content
-    });
-  };
-
+  const [altForm, setAltForm] = useState({ showAlt:false, meal:'', day:'', content:'' })
```

Comment:
remember you've renamed AltForm component to EditForm, you should do the same for the state as well